sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor{TPartition}.cs

```diff
@@ -1066,12 +1075,144 @@ async Task performProcessing()
         ///   load balancing between associated processors.
         /// </summary>
         ///
-        /// <param name="cancellationToken">A <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> instance to signal the request to cancel the operation.</param>
         ///
         private async Task RunProcessingAsync(CancellationToken cancellationToken)
         {
-            CreateConnection();
-            await Task.Delay(Timeout.Infinite, cancellationToken).ConfigureAwait(false);
+            cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
+
+            // Perform the tasks needed for the main processor loop, encompassing load balancing and management
+            // of processing for claimed partitions.
+            //
+            // Ensure that any errors observed, whether fatal or non-fatal, are surfaced to the exception handler in
+            // a fire-and-forget manner.  The processor does not assume responsibility for observing or surfacing exceptions
+            // that may occur in the handler, so the associated task is intentionally unobserved.
+
+            try
```

Comment:
This outer try/catch is intended to serve as a catch-all for scenarios that may otherwise crash the processor without invoking the error handler.   For example, the current processor would silently fail to start if creating a connection failed.  Outside of unexpected scenarios, it isn't intended that we bubble up to the outer level.